code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -356,7 +355,7 @@ can cause issues with ammo types getting mixed up during the burst.
 	accuracy_mult_unwielded = config.base_hit_accuracy_mult + config.low_hit_accuracy_mult - config.hmed_hit_accuracy_mult
 	scatter = config.med_scatter_value
 	scatter_unwielded = config.max_scatter_value
-	damage_mult = config.base_hit_damage_mult + config.high_hit_damage_mult
+	damage_mult = config.base_hit_damage_mult + config.low_hit_damage_mult
```

Comment:
This gun should have no damage bonus at all. Being able to PB twice with no cooldown is enough of a benefit.